### DISCUSSION TITLE
Grass distance limit seems to be stuck between 50-60 distance

### DISCUSSION THREAD

**Discussion by AlexandreHurbe** [visualdesigncafe/nature-renderer#34] [Help] [answered]
Hi, for some reason I cannot seem to increase the grass render distance, no matter what settings I use. You can see in the distance the bush is rendering and it's also using nature renderer.
![image](https://github.com/visualdesigncafe/nature-shaders/assets/37171772/c3f6ad84-796a-41d5-a430-d71a874246e1)

On the terrain, I have the "Nature Renderer" attached and the render distance is set to 250 for the grass:
![image](https://github.com/visualdesigncafe/nature-shaders/assets/37171772/5b16cb5d-ec29-493d-a212-3e249f7c2183)

On the camera, I have the "Nature Renderer Camera Settings" , with the settings below:
![image](https://github.com/visualdesigncafe/nature-shaders/assets/37171772/b804dbe8-1d1f-4727-aef8-d10175d7415b)

The grass prefab is never culled as well. I have tried with two different shaders including the nature shaders for grass but neither affect the distance.
![image](https://github.com/visualdesigncafe/nature-shaders/assets/37171772/df5c5ac8-d8f0-45a9-92b8-e6a65b54c985)

I have also tried editing the terrain detail distance without any luck. Within the editor, the grass also renders a further distance.

**Comment by CoreNut** [comment]
I have same issue. I can render grass much further when I disable Nature Renderer Component.

**Comment by maxplooi** [comment]
I am quite surprised to see this, all the settings look correct. Is it possible to share a small test project with just the terrain and the grass so that I can take a look at it?

**Comment by CoreNut** [comment, marked as answer]
Hello Max,
Much thanks for reply. I have been playing around with different shader
settings and it turned out that was the reason of culling.
Nature renderer is great asset, works perfect :)
Regards
Konrad

**Reply by maxplooi** [reply]
Awesome, glad to hear it is solved!
